### PR TITLE
release 5.0.0

### PR DESCRIPTION
**Pull request by Dan33l**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -264,4 +293,4 @@ This is the last release with Puppet3 support!
 Initial Release
 
 
-\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
+\* *This Changelog was automatically generated       by [github_changelog_generator]      (https://github.com/github-changelog-generator/github-changelog-generator)*
```
Comment (ekohl) [inline CHANGELOG.md:296]: What's this weird whitespace?

Comment (Dan33l) [reply CHANGELOG.md:296]: i don't know. i suppose it is from new version of generator

Comment (bastelfreak) [reply CHANGELOG.md:296]: is this reproducible? I have not seen this on other releases and the changelog generator gem was updated in thebpast half year.

Comment (Dan33l) [reply CHANGELOG.md:296]: when i run it twice i am getting same result.

Comment (Dan33l) [reply CHANGELOG.md:296]: The previous release of module was done in march 2020. The credit was modified in generator april 2020 : https://github.com/github-changelog-generator/github-changelog-generator/pull/794

Comment (ekohl) [reply CHANGELOG.md:296]: That's weird. It's almost as if `<<~CREDIT` isn't interpreted correctly. Which Ruby version are you using?

Comment (Dan33l) [reply CHANGELOG.md:296]: 2.5.3

Comment (ekohl) [reply CHANGELOG.md:296]: What does this return for you?
```console
$ bundle exec pry
/usr/share/ruby/irb/ruby-lex.rb:123: warning: already initialized constant RubyLex::ERROR_TOKENS
/usr/share/gems/gems/irb-1.3.5/lib/irb/ruby-lex.rb:123: warning: previous definition of ERROR_TOKENS was here
[1] pry(main)> require 'github_changelog_generator/generator/generator'
=> true
[2] pry(main)> GitHubChangelogGenerator::Generator::CREDIT_LINE
=> "\\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*\n"
```

As you can see, there are no odd spaces for me.

Comment (Dan33l) [reply CHANGELOG.md]: As you can see there are spaces for me :)
I manually removed them. No time for more.

Hohoho happy Christmas

**Review by smortex — APPROVED** [review]
No change when I run `bundle exec rake changelog` on top of these changes (after @Dan33l manual updates) with an up-to-date bundle.  LGTM.

**Review by ekohl — APPROVED** [review]

**Pull request merged**